### PR TITLE
fix: logout without redirects

### PR DESCRIPTION
**Pull request by winsonluk**
Clicking logout used to close the extension and open a new tab. This fix keeps the extension open and logs the user out seamlessly, without opening an extra tab.

### CODE REVIEW

File: extension/JavaScript/leaderboard.js [hunk 1]
```diff
@@ -2,13 +2,15 @@
 /*eslint no-undef: "error"*/
 
 /****************************************************************
-TODO/NOTES: 
-Where to implement clear_completed endpoint? --> only for testing 
+TODO/NOTES:
+Where to implement clear_completed endpoint? --> only for testing
```
Comment (hrmusicguy1) [inline extension/JavaScript/leaderboard.js:6]: is there a way to set this from the tests?

File: extension/JavaScript/leaderboard.js [hunk 2]
```diff
@@ -169,7 +177,7 @@ function populateTable(value, key) {
    cellMedalImage.style.height = "15px";
    cellMedalImage.style.marginTop = "3px";
 
-   let cellTeamLink = document.createElement("a");
```
Comment (hrmusicguy1) [inline extension/JavaScript/leaderboard.js:172]: haha so much extra whitespace

File: extension/JavaScript/leaderboard.js (hunk 1 above)
Comment (winsonluk) [reply extension/JavaScript/leaderboard.js:6]: i dont even know what this is, just fixing whitespace LOL

**Comment by Austin-Cheong** [comment]
Probably up to personal preference, but should there be a confirmation when the logout button is clicked? If not, then this looks good to merge.

**Comment by winsonluk** [comment]
> Probably up to personal preference, but should there be a confirmation when the logout button is clicked? If not, then this looks good to merge.

@Austin-Cheong Good idea, but most websites don't have logout confirmations, so it's prob better to follow their lead.

**Pull request merged**